draft-ietf-quic-transport.md

```diff
@@ -4984,6 +4999,12 @@ Sequence Number:
 : The sequence number assigned to the connection ID by the sender.  See
   {{issue-cid}}.
 
+Retire Prior To:
+
+: A variable-length integer specifying the limit for which all connection IDs
+  assigned to the receiver that have a sequence number less than this value
+  should be retired.  See {{retiring-cids}}.
```

Comment:
This sentence is kind of difficult to parse.

We should also specify that this value MUST be smaller or equal to the Sequence Number carried by this frame.

Reply:
The text below already species this must be <=. As improving the text, I agree it's a bit wordy. Do you have any specific suggestions for improving it?

Reply:
Given that the specifics of the limit are explained elsewhere, keep the field description brief.  "A variable-length integer indicating which connection IDs should be retired."